### PR TITLE
Deploy schedule

### PR DESCRIPTION
**Pull request by thibaultamartin**

### CODE REVIEW

File: templates/schedule.html [hunk 1]
```diff
@@ -22,31 +26,17 @@ <h1>Get your tickets now</h1>
         </div>
     </div>
     <div id="programme">
-        <div class="content-wrapper">
-            <div class="content">
-                <h2>Schedule</h2>
-                <table>
-                    <tr>
-                        <td>Thursday 19, September</td>
-                        <td>BarCamp</td>
```
Comment (HarHarLinks) [inline templates/schedule.html:31]: we might wanna keep this info. particularly the barcamp is entirely lost with the schedule embed

**Review by HarHarLinks — CHANGES_REQUESTED** [review]
~~this does not work on my phone at all~~ nope it's just weird because you can't really see the rooms. e.g. i guess we need to recommend the apps :D

File: sass/_programme.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#programme-content {
+    #programme {
+        overflow-x: scroll;
+    }
+}
```
Comment (HarHarLinks) [inline sass/_programme.scss:5]: something like this should fix the header pattern alignment (untested code!)
```suggestion
}

@media (min-width: 851px) {
  #programme-content {
    --pattern-height: 7;
  }
}
```

**Comment by MTRNord** [comment]
The overflow seems to be due to/tracked in https://github.com/pretalx/schedule/issues/17

I wasnt able to find an easy fix since its nested in their component which uses a shadow-root making it impossible to override from our side.

The .settings margin-left seems to push it over like that. Unsure why it does this though.

File: templates/schedule.html (hunk 1 above)
Comment (thibaultamartin) [reply templates/schedule.html:31]: It's lost in the schedule embed, but I'd rather add it in pretalx than to add the table

File: sass/_programme.scss (hunk 2 above)
Comment (thibaultamartin) [reply sass/_programme.scss:5]: Doesn't seem to fix, I'm not entirely sure what it does ._. ?
I think I'm going to leave it as is for now and see if we can fix it next week. I don't want to block the announcement on it

Comment (HarHarLinks) [reply sass/_programme.scss:5]: #50

**Pull request merged**